### ISSUE TITLE
Feature request - add IP Whitelist to tunnel

### ISSUE THREAD

**Issue by bklier-mpl** [ekzhang/bore#126]
Would it be possible to implement some command line magic like '--allow XXX.XXX.XXX.XXX/YY --allow ZZZ.ZZZ.ZZZ.ZZZ/AA ...' to just allow hosts matching the allowed networks to connect to the created tunnel? This would make it much easier to match security considerations while using this great tool to, for example, give remote access to remote support staff with known source addresses.

**Comment by ekzhang** [comment]
Yes, this would be possible! Not in scope for bore itself since it's meant to be minimal and for people to read the code and modify it to suit their use case.

**Comment by ekzhang** [comment]
I'm closing because additional options are not in scope for bore. It's a small program and meant to be read/forked if you have more particular use cases.

I don't have the resources to maintain a general-purpose tool, and I think something like this is better built for your specific use case regardless
